modules/compliance-profile-types.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * security/compliance_operator/co-concepts/compliance-operator-understanding.adoc
+// * security/compliance_operator/co-scans/compliance-operator-supported-profiles.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="compliance_profile_types_{context}"]
+= Compliance Operator profile types
+
+There are three types of compliance profiles available: Platform, Node, and {op-system-first} profiles. These profile types contain rules that check both OpenShift and operating system settings, such as {op-system} configurations. 

```

Comment:
This is a tricky thing to describe.

Technically - there are only two types of profiles. `Platform` and `Node`. But the ways in which we organize rules into profiles results in three different situations where you'd use a profile.

1.) Using a `Platform` profile to evaluate the OpenShift components, thus the platform
2.) Using a `Node` profile with OpenShift-specific rules to evaluate how OpenShift is configured across the entire cluster
3.) Using a `Node` profile with RHEL/RHCOS-specific rules to harden the underlying operating system running the OpenShift platform.

I'm not sure `{op-system}` is a clear replacement for the 3rd case though 🤔 .

Reply:
The three types makes me confusion. I like the suggestion here.